tasks/HelpMeCarry.tex

```diff
@@ -0,0 +1,76 @@
+\section{Help Me Carry [Party Host]}
+The robot helps a guest to carry some luggage to his/her car.
+
+\subsection{Main Goal}
+Robot helps a guest to carry a bag to the car which is parked outside and comes back home.
+
+\subsection{Focus}
+It includes understanding pointing gesture, object identification and manipulation, navigation in known and unknown environments, person-following while carrying a luggage, and social navigation.
+
+\noindent\textbf{Reward:} 500 pts for carrying the bag to the car's location and coming back home.
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Find the bag by understanding the pointing behaviour (100 pts).
+	\item Avoid all the obstacles outdoor (400 pts).
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The test takes place inside and outside the arena.
+
+	\item \textbf{Start Location}: The robot starts at a predefined location in the living room.
+
+	\item \textbf{Guest}: There is only one guest in the house, standing in front of the robot at the start.
+	
+	\item \textbf{Bags' Location:} There are at least two bags placed between one to two meters  away from the guest's start position and both should be visible to the robot.
+	
+	\item \textbf{Pointing to the Bag:} When the test starts, the guest points to his/her bag and requests the robot to carry it to the car.
+	
+	\item \textbf{Outdoor Obstacles:} The robot will face multiple obstacles outside. The obstacles may include a person, a group of people, a stationary object or a moving object.
```

Comment:
How many? Important to keep consistency. Avoiding a standing person is solved and shouldn't grant any points, while going through a crowd was typically awarded with 300.

Reply:
Ok then how about avoiding three obstacles 1) a crowd obstructing path 2) a small 3d object 3) a small toy car moving randomly in robot's path?

Reply:
Sounds good. Five obstacles were used in the past (3 from Follow Me, 2 from Navigation), but standing people and people walking by are kind of solved.

You can come out with some sort of new obstacle we haven't addressed before (e.g. traffic light). My only requirements are:
- Unspecified order
- Brief, concise description
- Nature and scoring clearly specified for each
- Special rule for collision if necessary

Reply:
Retractable Posts randomly placed in robot's path [like this](https://www.retaildisplaydirect.com.au/black-retractable-bollards/?gclid=CjwKCAiAqaTjBRAdEiwAOdx9xqzXfB8QDr1BujtBaUY280sbHXF9c7zBeBubXn9yR-syBUDteAx4aRoCJXYQAvD_BwE).

Or a random area covered with Barrier [tape](https://www.bollardshop.com.au/products/crowd-control/yellow-black-barrier-tape-75mm-x-100m/?gclid=CjwKCAiAqaTjBRAdEiwAOdx9xstZkVEZU78XDjh_ZKId9Cyj640E84OHxgY5_TlXLM1ibeb6qlKyAhoCvq4QAvD_BwE) and [bollards](https://www.safetyxpress.com.au/bollards/t-top-bollards-6kg-base/?gclid=CjwKCAiAqaTjBRAdEiwAOdx9xpmrHxkUiFaXdlaZr2fmy9lLuJFDLaVaciXw1LFmW_ZQLCJ7cG36lxoCfkcQAvD_BwE).

Those will cost under $100.

Reply:
None of those (yet). This is the nature of objects the robot had to avoid in the past.

![image](https://user-images.githubusercontent.com/8583949/52958416-54a84100-3394-11e9-984f-6889259632f8.png)
![image](https://user-images.githubusercontent.com/8583949/52958496-76a1c380-3394-11e9-9f1c-00950f744840.png)
![image](https://user-images.githubusercontent.com/8583949/52958551-9507bf00-3394-11e9-9df2-d83e0448468c.png)
![image](https://user-images.githubusercontent.com/8583949/52958629-c5e7f400-3394-11e9-9232-92e3f7d6f4f0.png)


We have had retractable posts but so far robots just undo their path (no optimization), so no robot has crashed yet with the ribbon, although I'm 100% sure no robot is able to see them.

### Constraints:
- A **Hard-to-see object** won't be detected by a Laser Rangefinder Scanner at a fixed height.
- A **Small object** is lying on the floor and tops 5cm tall. Normally humans don't see them, but robots must because of "reasons".
- An **interactable object** is either a person, rolling chair, door, or something the robot needs to interact with to get it/him/her out of its way.